.github/workflows/sycl-detect-changes.yml

```diff
@@ -81,8 +91,6 @@ jobs:
         uses: actions/github-script@v7
         with:
           script: |
-            console.log("Number of files changed:");
-            console.log(context.payload.pull_request.changed_files);
             if (context.payload.pull_request.changed_files < 500) {
```

Comment:
if we moved the `if` to the job level can we remove it here?

Reply:
We still need this to choose the outputs, unless we set the default output in 1st step too.